vai/__init__.py

```diff
@@ -89,9 +89,10 @@ def main():
         if args.dump_default_config:
             filename = models.Configuration.filename()
             if os.path.exists(filename):
-                print("Refusing to overwrite existing config file %s. Delete the file manually and try again." % filename)
-                sys.exit(1)
-
+                print("Operation will overwrite existing config file. Do you want to do this? (Y/n)", end=' ')
+                if input().lower() in ('no', 'n'):
```

Comment:
it's better if you check for yes, otherwise whatever thing you write will be taken as yes. Use the most conservative option as default.